### DISCUSSION TITLE
Feature request: Can we have a filter on the "list errors"

### DISCUSSION THREAD

**Discussion by ghalymt** [SubtitleEdit/subtitleedit#5804] [Ideas]
Can we have a filter on the "list errors" in tools, so we can choose the errors to be listed.
Thank you

**Comment by LeonCheung** [comment]
Something like #5476?

**Reply by ghalymt** [reply]
The new "list errors" option in "tools" are very useful in fixing errors manually faster. I am asking if it can be modified to just show the errors I want to fix manually "not all the errors".
